check/configuration/supported/clang-tidy.md

```diff
@@ -0,0 +1,72 @@
+---
+description: clang-tidy
+---
+
+# clang-tidy
+
+clang-tidy is a linter for C++.
+
+You can enable the clang-tidy plugin with
```

Comment:
I'm not sure why a bunch of these refer to it as the "${linter} plugin", but I think we should avoid that verbiage